apps/backend/src/modules/meetings/meeting-audio.model.ts

```diff
@@ -0,0 +1,34 @@
+import { type RelationMappings } from "objection";
+
+import {
+	AbstractModel,
+	DatabaseTableName,
+} from "~/libs/modules/database/database.js";
+
+import { MeetingAttribute } from "../meetings/libs/enums/enums.js";
+import { MeetingModel } from "../meetings/meetings.model.js";
+
+class MeetingAudioModel extends AbstractModel {
```

Comment:
Why do we need this? We need files table. In meetings we will have meetingAudioFileId